packages/db/src/schema/workspaces/validation.ts

```diff
@@ -3,11 +3,42 @@ import { z } from "zod";
 
 import { workspacePlans, workspaceRole } from "./constants";
 import { workspace } from "./workspace";
+import {
+  monitorFlyRegionSchema,
+  monitorPeriodicitySchema,
+} from "../shared/shared";
 
 export const workspacePlanSchema = z.enum(workspacePlans);
 export const workspaceRoleSchema = z.enum(workspaceRole);
 
+const limitsV1 = z.object({
```

Comment:
use the `@openstatus/plan` - we have defined here.

Reply:
but in openstatus plan have dep to openstatus/db 
`import type { WorkspacePlan } from "@openstatus/db/src/schema";` 
so we can't have dep for plan in openstatus/db but i want to validate for insert/select :) 
it's a circular dep it's 😱